tests/unit/univariate/test_student_t.py

```diff
@@ -0,0 +1,37 @@
+from unittest import TestCase
+
+import numpy as np
+from scipy.stats import t
+
+from copulas.univariate.student_t import StudentTUnivariate
+
+
+class TestStudentTUnivariate(TestCase):
+
+    def test__fit(self):
+        distribution = StudentTUnivariate()
+
+        data = t.rvs(size=50000, df=3, loc=1, scale=1)
+        distribution._fit(data)
+
+        expected = {
+            'df': 3,
+            'loc': 1,
+            'scale': 1,
+        }
+        for key, value in distribution._params.items():
+            np.testing.assert_allclose(value, expected[key], rtol=0.1)
+
+    def test__is_constant_true(self):
+        distribution = StudentTUnivariate()
+
+        distribution.fit(np.array([1, 1, 1, 1]))
+
+        assert not distribution._is_constant()
```

Comment:
Since now we will be also returning `True` if the distribution is truly constant, let's revert this to `assert distribution._is_constant()`.